src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java

```diff
@@ -4165,6 +4174,7 @@ private static final ExternalJavaScriptLoader getLoader(StaticDataResource relea
    private long lastModifiedTime_;
    private String yankedText_ = null;
    
+   private static AceEditor LAST_FOCUSED_EDITOR = null;
```

Comment:
Doesn't all-caps normally imply a constant?

Reply:
Oops, you're right. Should this still be `s_`-prefixed as we do in C++ sources?

Reply:
Our coding standards say yes, but seems to be very rare in the Java sources (I found one example).